packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -192,11 +204,161 @@ const getColSpanByKey = (
         }, 0);
 };
 
+const combinedRetrofit = (
+    data: PivotData,
+    getField: FieldFunction,
+    getFieldLabel: FieldLabelFunction,
+) => {
+    const indexValues = data.indexValues.length ? data.indexValues : [[]];
+    const baseIdInfo = last(data.headerValues);
+    const uniqueIdsForDataValueColumns: string[] = Array(
+        data.headerValues[0].length,
+    );
+
+    data.headerValues.forEach((headerRow) => {
+        headerRow.forEach((headerColValue, colIndex) => {
+            uniqueIdsForDataValueColumns[colIndex] =
+                uniqueIdsForDataValueColumns[colIndex] +
```

Comment:
```suggestion
                (uniqueIdsForDataValueColumns[colIndex] ?? '') + 
```
should this be

Reply:
Hah, I just suggested that above without realizing you wrote this.